### ISSUE TITLE
Objective C object literal syntax not recognized

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#578]
(Original issue 871 created by rogerbinns on 2013-05-19T03:20:59.694347+00:00)

As documented at [Clang](http://clang.llvm.org/docs/ObjectiveCLiterals.html) you can use @ for many many types of literals beyond the traditional strings.  However if you do so then pygments doesn't recognize it and refuses to highlight the code.  For example this is a valid line:

```
#!objc

id foo=@{@"string": @2, @"list": @[@"one", @2], @"expr": @(3+4)};
```

**Comment by Anteru** [comment]
(Original issue 871 last updated on 2013-05-19T06:55:57.454632+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2013-05-19T06:55:57.438714+00:00)

Closes #871: Objective C/C++ lexers: allow "@" prefixing any expression.

→ <<cset ad3a067a9881>>
